app-pda/libusbmuxd/libusbmuxd-1.0.10.recipe

```diff
@@ -0,0 +1,86 @@
+SUMMARY="A client library to multiplex connections from and to iOS devices"
+DESCRIPTION="libusbmuxd is a client library to multiplex connections from and \
+to iOS devices by connecting to a socket provided by a usbmuxd daemon"
+HOMEPAGE="http://www.libimobiledevice.org/"
+COPYRIGHT="2013-2014 Nikias Bassen
+	2009-2014 Martin Szulecki"
+LICENSE="GNU GPL v2
+	GNU LGPL v2.1"
+REVISION="1"
+SOURCE_URI="http://www.libimobiledevice.org/downloads/libusbmuxd-$portVersion.tar.bz2"
+CHECKSUM_SHA256="1aa21391265d2284ac3ccb7cf278126d10d354878589905b35e8102104fec9f2"
+PATCHES="libusbmuxd-$portVersion.patchset"
```

Comment:
Please remove PATCHES as well as the patchset file iteself.
("make LDFLAGS=-lnetwork" allows to avoid having a patchset.)